Laboratory/C++/run_test.sh

```diff
@@ -1,7 +1,15 @@
 #!/usr/bin/env bash
 set -e
 >&2 echo "Timing bebopc:"
-time dotnet run --project ../../Compiler --cpp "gen/$1.hpp" --files "../Schemas/$1.bop"
+
+if grep -q Microsoft /proc/version; then
+  # Windows: Visual Studio + WSL to run this script
+  bebopc="../../bin/compiler/Windows-Debug/bebopc.exe"
+else
+  # Linux or Mac
+  bebopc="dotnet run --project ../../Compiler"
+fi
+$bebopc --cpp "gen/$1.hpp" --files "../Schemas/$1.bop"
```

Comment:
should this still have a `time` command or no?

Reply:
I realized the time was not accurate anyway (on Linux/Mac, it includes the startup time of `dotnet` and compilation time, whereas I'm only really interested in the runtime).

Reply:
Oh shoot, the echo command above still says "Timing" though. Well, I'll fix it next time I run a test :)

Reply:
Oh shoot, the echo command above still says "Timing" though. Well, I'll fix it next time I run a test :)